src/dreammaker/lexer.rs

```diff
@@ -122,6 +124,7 @@ table! {
     "{\"", BlockString;
     "|",   BitOr;
     "|=",  BitOrAssign;
+    "||=",  SafeBitOrAssign;
```

Comment:
The meaning of "requires that [...] shorter entries be first" is that `||` must be before `||=` in the list